src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static DEFAULT_HEALTH = 100;
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = Animal.DEFAULT_HEALTH;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  bitten() {
```

Comment:
Better naming for this method would be `bite`.
Method name should usually be an answer to 'What does it do?' question. Here it answers 'What's the result after its execution?'

Reply:
I have bite method in Carnivore class. 
In this case, the animal is bitten, so I suppose the name should be healthReduce.

Thank you!

Reply:
Suggesting `bite` was my mistake (herbivore receives bites, but doesn't bite), so I've edited my comment with more appropriate names. Anyway, I see from your reply that you've got the point.